MobileCenter/MobileCenter/MSMobileCenter.m

```diff
@@ -353,6 +373,12 @@ - (void)sendStartServiceLog:(NSArray<NSString *> *)servicesNames {
   [self.logManager processLog:serviceLog withPriority:MSPriorityDefault andGroupID:kMSGroupID];
 }
 
+- (void)sendCustomPropertiesLog:(NSDictionary<NSString *, NSObject *> *)properties {
+  MSCustomPropertiesLog *customPropertiesLog = [MSCustomPropertiesLog new];
+  customPropertiesLog.properties = properties;
+  [self.logManager processLog:customPropertiesLog withPriority:MSPriorityDefault andGroupID:kMSGroupID];
```

Comment:
This will be a merge conflict with #444. `withProiority:MSPriorityDefault` need to be removed if #444 goes first.
Good to have TODO or FIXME